src/pages/Login/components/LoginBody.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+
+import { KAKAO_AUTH_URL } from './KakaoLoginAuth';
+
+import styled from 'styled-components';
+
+const LoginBody = () => {
+  const [userInputs, setUserInputs] = useState({
+    Id: '',
+    Pw: '',
+  });
+
+  const { Id, Pw } = userInputs;
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setUserInputs({
+      ...userInputs,
+      [name]: value,
+    });
+  };
+
+  const isInputValid = Id.length >= 2 && Pw.length >= 2;
+
+  return (
+    <LoginBodyWrapper>
+      <InputId onChange={handleInput} />
+      <InputPw onChange={handleInput} />
+      <SaveIdWrapper>
+        <SaveId />
+        아이디 저장
+      </SaveIdWrapper>
+      {isInputValid ? (
+        <LoginButton disabled={isInputValid}>로그인</LoginButton>
+      ) : (
+        <LoginButtonDisabled disabled={isInputValid}>
+          로그인
+        </LoginButtonDisabled>
+      )}
```

Comment:
LoginButton 컴포넌트와 LoginButtonDisabled 컴포넌트를 따로 만들어서 삼항연산자로 조건부 렌더링하는 것이 아니라, 하나의 LoginButton 컴포넌트의 disabled prop에 따라 해당 컴포넌트에서 조건부로 스타일링하는 것이 더 깔끔할 것 같습니다.
```suggestion
      <LoginButton disabled={isInputValid}>로그인</LoginButton>
```